src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -1491,13 +1506,21 @@ public void parseAdminNationCommand(String[] split) throws TownyException {
 					TownyMessaging.sendErrorMsg(getSender(), Translatable.of("msg_invalid_name"));
 
 			} else if (split[1].equalsIgnoreCase("merge")) {
+				NationCommand.mergeNation(player, StringMgmt.remArgs(split, 1), nation);

```

Comment:
```suggestion
				NationCommand.mergeNation(player, StringMgmt.remArgs(split, 2), nation);

```

Reply:
The mergeNation method expects the arguments `["merge", "<name>"]`, thats why its needed to include the `"merge"` too.